docs/docs/support/general.md

```diff
@@ -0,0 +1,52 @@
+---
+title: General
```

Comment:
@hifabienne regarding structure:  Can we call this page "Software release cycle"? and move the support states to https://zitadel.com/docs/legal/support-services ? 

But let's fix first the content comments.

Reply:
Note to self: clarify enterprise supported features in self-hosted and extended sla, the default (included/excluded), and how to become eligible

Reply:
Yes makes sense